projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/datepicker-docs/datepicker-docs.component.ts

```diff
@@ -166,6 +167,13 @@ export class DatepickerDocsComponent implements OnInit {
   }
   `;
 
+  resetDateExample: string = `
+  <go-datepicker
+    [control]="dobReset"
+    label="Date of Birth">
+  </go-datepicker>
```

Comment:
We really should show the JavaScript as well in the example. This HTML provided does not tell the implementor anything about _how to reset the control_.

Reply:
I have changed the same in our code.